src/PlacesAutocomplete.js

```diff
@@ -236,7 +236,12 @@ class PlacesAutocomplete extends Component {
         id="PlacesAutocomplete__root"
         style={this.inlineStyleFor('root')}
         className={this.classNameFor('root')}>
-        <input {...inputProps} />
+        <input {...inputProps} aria-describedby="PlacesAutocomplete__autocomplete-description"/>
+        <span 
+          id="PlacesAutocomplete__autocomplete-description"
+          style={this.inlineStyleFor('autocompleteDescription')}
```

Comment:
@bvellek We don't want regular (non-screen-reader) users to see this `span` tag. I think we should always use the default styles you defined in `defaultStyles.js` file. I don't see the use case where you need to override this style. What's your take on this?